pkg/deploy/generator/scripts/rpVMSS.sh

```diff
@@ -411,6 +419,14 @@ ExecStart=/usr/bin/docker run \
   --name %N \
   --rm \
   --cap-drop net_raw \
+  -e AZURE_FP_CLIENT_ID \
+  -e DOMAIN_NAME \
+  -e CLUSTER_MDSD_ACCOUNT \
+  -e CLUSTER_MDSD_CONFIG_VERSION \
+  -e GATEWAY_DOMAINS \
+  -e GATEWAY_RESOURCEGROUP \
+  -e MDSD_ENVIRONMENT \
+  -e CLUSTER_MDSD_NAMESPACE \
```

Comment:
I'm not sure if this is possible without a major refactor so might not make sense.  But these are unused (with the exception of the `AZURE_FP_CLIENT_ID` in the monitor.

Reply:
If the monitor component can't start without these variables but only one of them is actually used, it might be worth leaving a comment for cleanup when / if we get to the refactor.